templates/drop/drop-submitter_container.tmpl

```diff
@@ -39,15 +39,15 @@ singularity exec   \
 $AQUA_container  \
 bash -c \
 '
-cd /users/caprioli/AQUA/cli/lra
-aqua lra --config ifs-nemo-test.yaml -w $WORKERS -d -l INFO
+cd /users/caprioli/AQUA/cli/drop
```

Comment:
for being a template this is a bit wrong. Can we have some more generic than silvia's path?